src/views/app/folder-panel/parts/list-item-actions-wrapper.tsx

```diff
@@ -69,7 +69,7 @@ const HoverActionComponent = ({
 	const label = 'label' in action ? action.label : '';
 	const icon = 'icon' in action ? action.icon : '';
 	const onClick = useCallback(
-		(ev: SyntheticEvent<Element, Event> | KeyboardEvent): void => {
+		(ev): void => {
```

Comment:
removing the type will set ev type to any. is it intentional?

Reply:
It was a mistake. I forgot to put the right type